tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -103,6 +103,7 @@ const onlyOfficeCanvasEditorSelector = '#id_viewer_overlay'
 const onlyOfficeCanvasCursorSelector = '#id_target_cursor'
 const collaboraCanvasEditorSelector = '.leaflet-layer'
 const textEditorTextArea = '#text-editor-input'
+const filesActionDropdown = `//div[contains(@class, 'tippy-box')]//button[contains(@class, 'oc-files-actions-%s-trigger')]`
```

Comment:
```suggestion
const filesContextMenuAction = "#oc-files-context-menu button.oc-files-actions-%s-trigger"
```

Reply:
Also please, update this too https://github.com/owncloud/web/blob/0846a0a84ff57ae29c60de4d5cb327c9185761fe/tests/e2e/support/objects/app-files/resource/actions.ts#L40
to
```js
const filesBatchAction = ".files-app-bar-actions .oc-files-actions-%s-trigger"
```